pkgs/development/python-modules/responses/default.nix

```
@@ -1,16 +1,27 @@
-{ buildPythonPackage, fetchPypi
-, cookies, mock, requests, six }:
+{ buildPythonPackage, fetchPypi, lib, python3Packages }:
```

while in python-modules, please import the packages individually:
```suggestion
{ buildPythonPackage, fetchPypi, lib, cookies, mock, requests, six, flake8, pytest, pytestcov, pytest-localserver }:
```